extension/src/popup/components/sendPayment/SendAmount/index.tsx

```diff
@@ -290,8 +292,13 @@ export const SendAmount = ({
   ]);
 
   // for swaps we're loading and choosing the default destinationAsset here
+  // also, need to check if both source and destination are native
```

Comment:
This is kind of curious, why would we allow a swap from native to native?

Reply:
The tests would sometimes fail without this check because both the source and destination would be set to XML. I noticed it in the app as well when refreshing the page when on that view. I'm not sure if this is the best fix, but it seemed the least risky.

Reply:
Hmm yeah fair enough, I think it's probably an existing bug if we can setup a swap where both sides are the same asset but the check doesn't make it worse so I'll go hit the approve button, thanks!